src/side-navigation/side-navigation.scss

```diff
@@ -2,9 +2,10 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import '../style/index';
 @import '../icon/index';
+@import '../text/index';
 
 @mixin iui-side-navigation {
-  display: inline-flex;
+  display: flex;
```

Comment:
Why changed?

Reply:
I was having issues placing the sub-menu correctly, but I must have fixed that elsewhere and this change was unneeded.  Reverting.

Reply:
Forgot to push? 😁

Reply:
Yeah, sorry, having a rough start back from moving and forgetting things. 😅 I'm going to stop commenting "fixed in the next push" and only comment after it is pushed.